client/src/pages/Login.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import Link from '@material-ui/core/Link';
+import Grid from '@material-ui/core/Grid';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import InputBase from '@material-ui/core/InputBase';
+import Divider from '@material-ui/core/Divider';
+
+import Logo from '../images/logo.png';
+
+const BootstrapInput = withStyles((theme) => ({
+    root: {
+        'label + &': {
+            marginTop: theme.spacing(3),
+        },
+    },
+    input: {
+        borderRadius: 4,
+        position: 'relative',
+        backgroundColor: theme.palette.common.white,
+        border: '1px solid #ced4da',
+        fontSize: 16,
+        width: '100%',
+        height: 22,
+        padding: '10px 12px',
+        textAlign: 'center',
+        transition: theme.transitions.create(['border-color', 'box-shadow']),
+    },
+}))(InputBase);
```

Comment:
In general use `MaterialUI TextField` instead of `InputBase` to have a much easier time

Reply:
True, but I was trying to make the `input` as similar as the mockup, the `TextField` is too to ‘thick’ and kinda too complicated to customize. That's the reason why I was using a `InputBase`  with customized Bootstrap style. Is that okay to keep it as now or should I change to TextField and just keep it simple as a thick input?

Reply:
That's fine. Just in the future!